ppa/archive/management/commands/index_pages.py

```diff
@@ -0,0 +1,106 @@
+'''
+Custom multiprocessing Solr index script for page index data.
+'''
+
+import queue
+from multiprocessing import Process, Queue, cpu_count
+from time import sleep
+
+from django.core.management.base import BaseCommand
+from parasolr.django import SolrClient, SolrQuerySet
+import progressbar
+
+from ppa.archive.models import DigitizedWork, Page
+
+
+def page_index_data(work_q, page_data_q):
+    '''Function to generate page index data and add it to
+    a queue. Takes a queue with digitized works to generate pages
+    for and a queue where page data will be added.'''
+    while True:
+        try:
+            digwork = work_q.get(True, 1)
+            # convert to the generator to a list
```

Comment:
```suggestion
            # convert the generator to a list
```
Do we really care about nitpicking stuff like this?

Reply:
I appreciate the edits and corrections if you spot them — anything that makes it more readable later is always better.